### DISCUSSION TITLE
Artemis Codes & RPCS3

### DISCUSSION THREAD

**Discussion by CorpseSlayer** [bucanero/ArtemisPS3#78] [Q&A]
I am so sorry if this is not the place to ask but i totally need to ask this question maybe i can make this codes work some how for RPCS3

something simple
[Atemis ]

Infinite Health
0
CodeFreak/PS3UserCheat
6 01140844 0000001C
6 00000000 0000104C
0 00000000 47F42400
6 01140844 0000001C
6 00000000 00001048
0 00000000 47F42400

that simple code in RPCS3 is this:

PPU-506c11110b75cb60e70ed04b81909118dc52b882:
  "Infinite Health":
    Games:
      "Hard Corps Uprising (Artemis)":
        NPEB00332: [ 01.00]
    Author: CodeFreak/PS3UserCheat
    Notes: 
    Patch Version: 1.2
    Patch:
      - [ be32,0x01140844,0x0000001C]
      - [ be32,0x00000000,0x0000104C]
      - [ be32,0x00000000,0x47F42400]
      - [ be32,0x01140844,0x0000001C]
      - [ be32,0x00000000,0x00001048]
      - [ be32,0x00000000,0x47F42400]
      
in the Artemis code [6 01140844 0000001C] i use 0x01140844,0x0000001C but the code wont work after a finish doing the list
so i was wondering is the 6 important to make the code work and how can i add the missing 6 to RPCS3 list or this code, for now i have in a YML file with around 100 games codes using Artemis codes i just need little help on how to figure things out.

**Comment by bucanero** [comment]
Type 6 is a pointer code. You should check the NetCheat code reference:
https://github.com/bucanero/ArtemisPS3/wiki/Artemis-NetCheat-code-types

then see if there's a way to implement all the code types on your emulator.
